lego/apps/users/migrations/0039_penalty_activation_date.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.10 on 2023-04-26 09:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("users", "0038_alter_abakusgroup_type"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="penalty",
+            name="activation_date",
+            field=models.DateTimeField(
+                default=None, null=True, verbose_name="date created"
+            ),
+        ),
```

Comment:
Doesn't work for active penalties. I think just running `save()` on all active penalties should fix it